lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -143,8 +143,8 @@ def main():
         catB_time_correction = np.array(catB_calib["time_correction"])
         catB_unusable_pixels = np.array(catB_calib["unusable_pixels"])
 
-        # set unusable all pixels in periods with too many unusable pixels (car flashes, etc...)
-        catB_unusable_pixels[np.sum(catB_unusable_pixels, axis=2) > args.max_unusable_pixels] = True
+        # add good time interval column (gti)
+        catB_calib['gti'] = np.max(np.sum(catB_unusable_pixels, axis=2),axis=1) < args.max_unusable_pixels
```

Comment:
A GTI is a good time interval. However, this is more a flag that this particular monitoring point "is valid" or has a general problem.

You would build the DL3 GTI by looking at all possible sources of issues (including the calibration monitoring data), but I wouldn't call this GTI here. Maybe just "is_valid"? We have `xy_is_valid` in many places in the ctapipe data structures.